src/Tvheadend.cpp

```diff
@@ -1060,6 +1060,22 @@ PVR_ERROR CTvheadend::DeleteTimer
     /* EPG-query-based repeating timer */
     return m_autoRecordings.SendAutorecDelete(timer);
   }
+  else if ((timer.iTimerType == TIMER_ONCE_CREATED_BY_TIMEREC) ||
+           (timer.iTimerType == TIMER_ONCE_CREATED_BY_AUTOREC))
+  {
```

Comment:
I wonder if " UpdateTimer" works correct as we don't use "TIMER_ONCE_CREATED_BY_XY" there anywhere. And still have to update the enabled state for tvh 4.1 (also for read only)?